packages/design-system/docs/components/OcAvatar/OcAvatar.md

```diff
@@ -0,0 +1,31 @@
+# OcAvatar component
+
+## Description
+
+The `OcAvatar` component is a thumbnail that is usually representing a user or a group. It can either be an image or the initials of a given username. There are a few variants of the component that can be used in specific contexts. See the examples below.
```

Comment:
for all the individual avatar components you could add another paragraph to the end of the description, stating `If you want to use multiple avatar items in one container, have a look at <link to OcAvatars docs>`